src/core/sdk.js

```diff
@@ -70,6 +70,12 @@ export default class SDK {
   getToken() {
     return this.storage.getItem(this.options.accessTokenKey)
       .then(access_token => {
+        if (!access_token) {
+          this.tokenRequestPromise = null;
+
+          return Promise.reject(new AuthorizationRequiredError());
```

Comment:
We don't really need to reject it with the `AuthorizationRequiredError ` since we're already doing it on the `catch` below..